### PR TITLE
[Rename]-Positive name for SSL flag

### PR DESCRIPTION
**Pull request by jignesh-crest**
## Closes https://github.com/elastic/connectors-py/issues/550

This PR addresses the change related to renaming the ssl flag, by changing `ssl_disabled` -> `ssl_enabled`. 

#### Pre-Review Checklist
- [x] Covered the changes with automated tests
- [x] Tested the changes locally
- [x] Added a label for each target release version (example: `v7.13.2`, `v7.14.0`, `v8.0.0`)
- [x] Considered corresponding documentation changes
- [ ] ~~Contributed any configuration settings changes to the configuration reference~~

### CODE REVIEW

File: connectors/utils.py [hunk 1]
```diff
@@ -399,3 +401,74 @@ async def func_to_execute(*args, **kwargs):
         return func_to_execute
 
     return wrapper
+
+
+def ssl_context(certificate):
```
Comment (timgrein) [inline connectors/utils.py:406]: Moving these methods and adding new ones should be separate PR. Let's focus this one only on `ssl_disabled` -> `ssl_enabled`

Comment (jignesh-crest) [reply connectors/utils.py]: Removed the unrelated changes form the PR. You can take a look.

**Comment by moxarth-elastic** [comment]
buildkite test this

**Review by timgrein — APPROVED** [review]
LGTM, thank you for the changes!

**Comment by moxarth-elastic** [comment]
buildkite test this
